docs/index.md

```diff
@@ -0,0 +1,99 @@
+---
+organization: Turbot
+category: ["saas"]
+icon_url: "/images/plugins/turbot/circleci.svg"
+brand_color: "#41667E"
+display_name: "CircleCI"
+name: "circleci"
+description: "Steampipe plugin for querying resource projects, pipelines, builds and more from CircleCI."
+og_description: "Query CircleCI with SQL! Open source CLI. No DB required."
+og_image: "/images/plugins/turbot/circleci-social-graphic.png"
+---
+
+# CircleCI + Steampipe
+
+[CircleCI](https://www.circleci.com/) is the leading open source automation server, CircleCI provides hundreds of plugins to support building, deploying and automating any project.
+
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query cloud APIs using SQL.
+
+For example:
+
+```sql
+select
+  username as "organization",
```

Comment:
Does the `username` column correspond to the name of the organization or the name of the user? The column description is a bit confusing.

Reply:
I don't like this column name as well, but it's how Circle CI API calls this property. It represents the VCS username where the repository is in. Such as  `turbot/steampipe` where turbot is the VCS username, as well as the organization.